schema/proto3/router.proto

```diff
@@ -32,13 +37,19 @@ message RouterConfiguration {
     MessageType message_type = 5; // DELTA (default) or FULL
     string host_dvr_mac_address = 6;
 
+    message RoutingRuleExtraInfo{
+        DestinationType destination_type = 1;
+        string next_hop_mac = 2;
+        }
+
     message RoutingRule {
         OperationType operation_type = 1;
         string id = 2;
         string name = 3;
-        string destination = 4;
-        string nextHop = 5;
+        string destination = 4;  // destination IP, could be 154.12.42.24/32 (host address) or 0.0.0.0/0 (network address)
+        string next_hop_ip = 5;
```

Comment:
Do we expect this has to be IP, or could be others like a label?

Reply:
this is expected to be IP only, confirmed with James this morning.

Reply:
Yes. "nexthop" should be an IP address of a gateway, but we can use "nexthop_type" to indicate what type of the nexthop or gateway, such as "local", "vpc", "internet", "ngw", etc...

Reply:
should it be nexthop_type or destination_type existed in RoutingRuleExtraInfo? 

I think destination_type is more appropriate since it is more important about the destination.